config.rb

```diff
@@ -57,4 +57,7 @@ def self.development?
   def self.test?
     Config.rack_env == "test"
   end
+
+  override :hetzner_user, string
+  override :hetzner_password, string
```

Comment:
should have a clear: true on this. Also are you sure you mean override or 'optional'?